### DISCUSSION TITLE
Seeing extra square brackets with Quarto to docx export

### DISCUSSION THREAD

**Discussion by jackflorey** [quarto-dev/quarto-cli#10421] [Q&A] [answered]
### Description

I'm using Quarto version 1.5.54 on OS X 14.5. 

When I render a simple qmd file to docx, I see extra square brackets enclosing each section in the docx file as seen in the attached image. qmd file is shown below. Any ideas on what I'm doing wrong?

```qmd
---
title: "Quarto Word Export Test"
format:
  docx:
    toc: true
    number-sections: true
    highlight-style: github
    number-depth: 3
---

# One
Introduction goes in this section.

# Two
This is the middle.

# Three
Let's conclude here.
```

<img width="504" alt="quarto-test" src="https://github.com/user-attachments/assets/95e23591-7d4b-4c4d-a581-5a9831c48957">

**Comment by mcanouil** [comment, marked as answer]
The gray brackets are a Microsoft Word feature (about editable text / bookmark from what I remember) which is unrelated to Quarto.
They won't show in prints or when saved as PDF.

**Reply by jackflorey** [reply]
Thank you! I looked into this after your comment, and found that unchecking "Bookmarks" in "View" preferences solves the issue. Leaving a screenshot here for others.
<img width="1118" alt="quarto-remove-bookmarks" src="https://github.com/user-attachments/assets/c18e7341-dcdb-4d6a-b90d-02dff08b1af1">

**Reply by jeffkimbrel** [reply]
I think it is still related to Quarto in that it is somehow creating/generating these bookmarks. It looks like it may be a linking feature, in case your document needs to internally link to a different section. If you're like me and you send a quarto generated .docx to a collaborator and they don't know what those brackets are, they can be deleted (bracket by bracket) in the "insert -> bookmark" menu within Microsoft Word.
